devlab/utils/os_cli.sh

```diff
@@ -146,7 +150,7 @@ function delete_volume {
 }
 
 function get_volume_list {
-  ${ssh} nova volume-list --all-tenants | grep none | cut -d ' ' -f 2
+  ${ssh} nova volume-list --all-tenants | grep available | cut -d ' ' -f 2
```

Comment:
This skips `in-use` volumes, is this expected?... I'm assuming this function would return *all* volumes, so perhaps `cinder list --all-tenants | sed -nr 's/^\| (\w{8}-\w{4}-\w{4}-\w{4}-\w{12}).*/\1/p'` would do the job correctly.